site/en/docs/privacy-sandbox/attribution-reporting-debugging/part-2/index.md

```diff
@@ -37,11 +37,19 @@ Check that you've applied best practices for API integration
 
 {% endDetailsSummary %}
 
-- Check that your code is gated behind [feature detection](https://docs.google.com/document/d/1BXchEk-UMgcr2fpjfXrQ3D8VhTR-COGYS1cwK_nyLfg/edit#heading=h.7yomtnn3mrs4).
+- Check that your code is gated behind feature detection. Before you use the API, detect if it hasn't been disallowed on the page via a Permissions-Policy. To do so, run the following code:
```

Comment:
"Before you use the API, check if it hasn't been blocked from the page because of a Permissions-Policy."

This one's a suggestion that I think is easier to understand—the via should definitely go, it makes it easier to misread, the "blocked" i'm less sure about. Might be worth checking with Rachel, perhaps?